appscale/tools/appscale_tools.py

```diff
@@ -62,11 +68,39 @@ class AppScaleTools(object):
   # A regular expression that matches files compressed in the zip format.
   ZIP_REGEX = re.compile(r'.zip\Z')
 
+
   # A str that contains all of the authorizations that an AppScale cloud
   # administrator should be granted.
   ADMIN_CAPABILITIES = "upload_app"
 
 
+  # AppScale repository location on local filesystem.
+  APPSCALE_REPO = "~/appscale"
+
+
+  # Command to run the Bootstrap.
+  RUN_BOOTSTRAP_COMMAND = "bash bootstrap.sh"
+
+
+  # String of commands to cd into the AppScale repository and run the Bootstrap.
+  BOOTSTRAP_COMMAND = "cd " + APPSCALE_REPO + ";" + RUN_BOOTSTRAP_COMMAND
+
+
+  # Command to run the upgrade script from /appscale/scripts directory.
+  UPGRADE_SCRIPT = "python " + APPSCALE_REPO + "/scripts/datastore_upgrade.py"
+
+
+  # Data Upgrade Success keyword from log file.
+  DATA_UPGRADE_SUCCESS = "Data upgrade status: SUCCESS"
```

Comment:
Same here. I wouldn't check for a specific string related to a data upgrade since later upgrades won't necessarily be data-related. One option is to write a response on the server side in JSON format that includes the success status and a more specific message if needed.